src/cpuid/src/lib.rs

```diff
@@ -625,3 +625,32 @@ impl From<RawKvmCpuidEntry> for (CpuidKey, CpuidEntry) {
         )
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn get() {
+        let cpuid = Cpuid::Intel(IntelCpuid(std::collections::BTreeMap::new()));
+        assert_eq!(
+            cpuid.get(&CpuidKey {
+                leaf: 0,
+                subleaf: 0
+            }),
+            None
+        );
+    }
+
+    #[test]
+    fn get_mut() {
+        let mut cpuid = Cpuid::Intel(IntelCpuid(std::collections::BTreeMap::new()));
+        assert_eq!(
+            cpuid.get_mut(&CpuidKey {
+                leaf: 0,
+                subleaf: 0
+            }),
+            None
+        );
+    }
+}
```

Comment:
This test looks duplicate of ones under `src/cpuid/src/intel/mod.rs`. Do we need this?

Reply:
The functions being tested here `Cpuid::get` and `Cpuid::get_mut` do call `IntelCpuid::get` and `IntelCpuid::get_mut` but this is testing additional functionality.